Week3/homework/step2-3.js

```diff
@@ -30,9 +36,13 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  let i = 0;
+  if (num > 0) {
```

Comment:
Here, the test is indeed needed. This is because a `do-while` loop tests the loop condition at the end of the loop and the loop body will always be executed at least once if precautions are not taken.